README.md

```diff
@@ -244,6 +244,7 @@ See also [Rust - Production](https://www.rust-lang.org/production) organizations
 * [Hiqlite](https://github.com/sebadob/hiqlite) - highly-available, embeddable, raft-based SQLite + cache
 * [indradb](https://crates.io/crates/indradb) - Graph database
 * [lancedb](https://github.com/lancedb/lancedb) [[vectordb](https://crates.io/crates/vectordb)] - A serverless, low-latency vector database for AI applications
+* [Limbo](https://github.com/tursodatabase/limbo) - Limbo is a work-in-progress, in-process OLTP database management system, compatible with SQLite. ![Build Status](https://github.com/penberg/limbo/actions/workflows/rust.yml/badge.svg)
```

Comment:
Build status is for a different username

Reply:
Removed it